### ISSUE TITLE
Position of `ListEmptyComponent` 

### ISSUE THREAD

**Issue by alexpchin** [PedroBern/react-native-collapsible-tab-view#21]
I'm just investigating the rendered position when you include a `ListEmptyComponent`. I believe the position is correct when you scroll to hide the header (i.e. the List EmptyComponent is in the middle of the screen). However, it displays too far down when the header is displayed (the default on page load).

For example in `Contacts.tsx`.

```js
const ListEmptyComponent = () => (
  <View
    style={{
      alignItems: 'center',
      flex: 1, // Without it will dispay at the very top
      justifyContent: 'center',
    }}
  >
    <Text>No results.</Text>
  </View>
);

export default class Contacts extends React.Component {
  render() {
    return (
      <FlatList
        data={CONTACTS}
        keyExtractor={(_, i) => String(i)}
        renderItem={renderItem}
        ItemSeparatorComponent={ItemSeparator}
      />
    );
  }
}

// used in Collapsible TabView examples
export const AnimatedContacts = React.forwardRef<
  any,
  React.PropsWithoutRef<CollapsibleScenePropsAndRef>
>((props, ref) => {
  const [isRefreshing, startRefreshing] = useRefresh();

  return (
    <Animated.FlatList
      ref={ref}
      data={CONTACTS}
      keyExtractor={(_, i) => String(i)}
      renderItem={renderItem}
      ItemSeparatorComponent={ItemSeparator}
      ListEmptyComponent={ListEmptyComponent} // <--- Here
      refreshing={isRefreshing}
      onRefresh={startRefreshing}
      {...props}
    />
  );
});
```

![Screenshot 2020-12-08 at 13 25 25](https://user-images.githubusercontent.com/40461/101489459-e1046900-3958-11eb-9b52-4c902e287817.png)

And

![Screenshot 2020-12-08 at 13 25 34](https://user-images.githubusercontent.com/40461/101489470-e5c91d00-3958-11eb-947a-7c4a26a5845f.png)

**Comment by alexpchin** [comment]
Can we perhaps resolve this using `flexGrow: 1` instead of using minHeight?

```js
export const AnimatedContacts = React.forwardRef<
  any,
  React.PropsWithoutRef<CollapsibleScenePropsAndRef>
>(({ contentContainerStyle, ...rest }, ref) => {
  const [isRefreshing, startRefreshing] = useRefresh();

  const contentContainerStyleProp = {
    paddingTop: contentContainerStyle.paddingTop || 0,
    flexGrow: 1,
  };

  return (
    <Animated.FlatList
      ref={ref}
      data={CONTACTS}
      keyExtractor={(_, i) => String(i)}
      renderItem={renderItem}
      ItemSeparatorComponent={ItemSeparator}
      ListEmptyComponent={ListEmptyComponent}
      refreshing={isRefreshing}
      onRefresh={startRefreshing}
      contentContainerStyle={contentContainerStyleProp}
      {...rest}
    />
  );
});
```

Gives: 

![Screenshot 2020-12-08 at 13 38 43](https://user-images.githubusercontent.com/40461/101490756-b74c4180-395a-11eb-93de-6268d3a93d13.png)

**Comment by PedroBern** [comment]
@alexpchin I'm not with the project open right now, I will check it later, but I think removing the `minHeight` will re-introduce a bug in case of the content being smaller than the screen height (#8). Can you check this by adding the same `contentContainerStyleProp` to the `SmallContentExample` ?

**Comment by alexpchin** [comment]
@PedroBern Yes, you are correct. It does create a gap at the top when scrolling between tabs. I will look into it further now.

**Comment by PedroBern** [comment]
@alexpchin check out the #22 preview, please let me know if it is working for you or if you have something to add. When you say it's ok I will merge and release :)

**Comment by alexpchin** [comment]
@PedroBern is the man. 🚀

**Comment by PedroBern** [comment]
released in `v1.4.1` 🚀

**Comment by alexpchin** [comment]
@PedroBern If there were two EmptyList Components:

i.e. Very simplified example to reproduce:

```js
.
.
.

const renderScene = SceneMap({
  contacts: ContactsScene,
  contacts2: ContactsScene,
});

const App: React.FC<object> = () => {
  const [index, setIndex] = React.useState(0);
  const [routes] = React.useState<Route[]>([
    { key: 'contacts', title: 'Contacts' },
    { key: 'contacts2', title: 'Contacts2' },
  ]);

.
.
.
```

With the current implementation, you would get:

<img width="491" alt="Screenshot 2020-12-09 at 17 10 27" src="https://user-images.githubusercontent.com/40461/101662532-885ccb00-3a41-11eb-8c67-1250e263c6b4.png">

And:

<img width="461" alt="Screenshot 2020-12-09 at 17 10 53" src="https://user-images.githubusercontent.com/40461/101662547-8d217f00-3a41-11eb-9f69-d704150d7810.png">
